.concourse/suse-buildpacks-ci/buildpack-version-bump/tasks/create_pr.sh

```diff
@@ -40,7 +40,7 @@ git checkout -b "${GIT_BRANCH_NAME}"
 
 perl -i -0pe "s/      ${BUILDPACK_NAME}:\n        version: \"[\d.]+\"/      ${BUILDPACK_NAME}:\n        version: \"${RELEASE_VERSION}\"/" "${KUBECF_VALUES}"
 
-git commit "${KUBECF_VALUES}" -m "${COMMIT_TITLE}"
+git commit "${KUBECF_VALUES}" -m "${COMMIT_TITLE}" || exit 0
```

Comment:
If we just want to not do this if nothing has changed, we should be using `git status --porcelain` instead; there are other situations where we'd fail to commit, and we _do_ want to know about those.